Let depmod be called by SymlinkProgram instead of inside the makefile.

diff -Naur fritz/src/Makefile fritz.new/src/Makefile
--- fritz/src/Makefile	2007-07-05 15:00:00.000000000 +0200
+++ fritz.new/src/Makefile	2008-02-08 15:47:00.000000000 +0100
@@ -47,7 +47,6 @@
 install:	
 		@if ! test -d $(INSTALLDIR); then mkdir -p $(INSTALLDIR); fi
 		@cp -vf fwlanusb.ko $(INSTALLDIR)
-		depmod -a
 
 endif
 
